qiskit/circuit/quantumcircuit.py

```diff
@@ -1917,6 +1925,43 @@ def _rebind_definition(self, instruction, parameter, value):
                             op.params[idx] = param.bind({parameter: value})
                         self._rebind_definition(op, parameter, value)
 
+    def convert_durations_to_dt(self, dt_in_sec, inplace=True):
```

Comment:
why is this a method on the QuantumCircuit class? can't we make this a utility function somewhere else?

Reply:
Yes, we can. I'll move it to `circuit/duration.py`.